mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -10032,6 +10105,30 @@ func testAccContainerCluster_withWorkloadALTSConfig(projectID, name, networkName
 	}
 `, projectID, networkName, subnetworkName, name, enable)
 }
+
+func testAccContainerCluster_withWorkloadALTSConfigAutopilot(projectID, name string, enable bool) string {
+	return fmt.Sprintf(`
+	data "google_project" "project" {
+		provider = google-beta
+		project_id = "%s"
+	}
+	resource "google_container_cluster" "with_workload_alts_config" {
+		provider = google-beta
+		name               = "%s"
+		location           = "us-central1"
+		initial_node_count = 1
+		workload_alts_config {
+			enable_alts = %v
+		}
+		workload_identity_config {
+			workload_pool = "${data.google_project.project.project_id}.svc.id.goog"
+		}
+		enable_autopilot = true
+		deletion_protection = false
+	}
```

Comment:
only nit - we use spaces for indentation for Terraform configs

Reply:
Fixed.